source/agora/node/Ledger.d

```diff
@@ -390,7 +390,7 @@ public class Ledger
         if (data.enrolls.length + active_enrollments < Enrollment.MinValidatorCount)
             return "Enrollment: Insufficient number of active validators";
 
-        foreach (enroll; data.enrolls)
+        foreach (const ref enroll; data.enrolls)
```

Comment:
Move to the second commit right ?

Reply:
The 2nd commit was for `inputs` only and I was thinking of keeping everything else in the 1st commit.
Since this is `enroll` and not related to `tx`, it might be a good idea to move it.
What do you think?

Reply:
How about applying the title and contents of the second commit comprehensively in the revision?